src/core/api/project.go

```diff
@@ -444,8 +447,11 @@ func (p *ProjectAPI) populateProperties(project *models.Project) error {
 		project.SetMetadata(models.ProMetaSeverity, strings.ToLower(vuln.ParseSeverityVersion3(severity).String()))
 	}
 
-	if p.SecurityCtx.IsAuthenticated() {
-		roles := p.SecurityCtx.GetProjectRoles(project.ProjectID)
+	if sc, ok := p.SecurityCtx.(*local.SecurityContext); ok {
```

Comment:
Is this intentional? Previously it doesn't require the securityContext must be local based.